Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -51,6 +51,7 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         tableView.layer.cornerRadius = 10
         tableView.separatorStyle = .None
         tableView.cellLayoutMarginsFollowReadableWidth = false
+        tableView.accessibilityLabel = "Context Menu"
```

Comment:
If you don't want this to be a user-facing accessibility string. Use accessibilityIdentifier instead.